src/MockedData.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ *
+ */
+
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+
+import { DateFromString } from "italia-ts-commons/lib/dates";
+import { NonNegativeNumber } from "italia-ts-commons/lib/numbers";
+
+import { CreditCard, CreditCardNumber } from "./types/CreditCard";
+import {
+  PaymentMethod,
+  PaymentMethodEnum,
+  PaymentMethodType
+} from "./types/PaymentMethod";
+import { CountryCurrencyCode, Transaction } from "./types/Transaction";
+import { User } from "./types/User";
+
+/**
+ * Generate Credit Card
+ * @param {string} id
+ * @param {NonEmptyString} issuer
+ * @param {CreditCardNumber} num
+ * @param {string} message
+ * @returns {CreditCard}
+ */
+function genCC(
```

Comment:
use more verbose names for functions, name should be descriptive (e.g. `createCreditCard` or `newCreditCard`)